doctr/utils/geometry.py

```diff
@@ -127,11 +127,41 @@ def rotate_abs_geoms(
     return rotated_polys
 
 
+def remap_boxes(
+    loc_preds: np.ndarray,
+    orig_shape: Tuple[int, int],
+    dest_shape: Tuple[int, int]
+) -> np.ndarray:
+    """ Remaps a batch of rotated locpred (x, y, w, h, alpha, c) expressed for an origin_shape to a destination_shape.
```

Comment:
Is this a typo in the docstring ? The function should work with the new box format [[x1, y1], [x2, y2], ...]]

Reply:
typo indeed